packages/local_auth/local_auth/lib/local_auth.dart

```diff
@@ -12,14 +12,14 @@ import 'dart:async';
 
 import 'package:flutter/foundation.dart' show visibleForTesting;
 import 'package:flutter/services.dart';
+import 'package:local_auth_platform_interface/local_auth_platform_interface.dart';
+import 'package:local_auth_platform_interface/types/auth_messages.dart';
+import 'package:local_auth_platform_interface/types/auth_options.dart';
+import 'package:local_auth_platform_interface/types/biometric_type.dart';
 import 'package:platform/platform.dart';
-
 import 'auth_strings.dart';
-import 'error_codes.dart';
-
-enum BiometricType { face, fingerprint, iris }
 
-const MethodChannel _channel = MethodChannel('plugins.flutter.io/local_auth');
+export 'package:local_auth_platform_interface/types/biometric_type.dart';
```

Comment:
We should remember to make the version update a breaking change when re-publishing this plugin, since this is where we're introducing the new enum values. (Which doesn't necessarily *have* to be a breaking change, but I think that's the best way to do it.)